server/types/oid/iterate.go

```diff
@@ -685,12 +765,16 @@ func runCallbackValidation(ctx *sql.Context, oid uint32, callbacks Callbacks) bo
 		if callbacks.Table == nil {
 			return false
 		}
+	case Section_Type:
+		if callbacks.Type == nil {
+			return false
+		}
 	case Section_View:
 		if callbacks.View == nil {
 			return false
 		}
 	default:
-		return false
+		return true
```

Comment:
This should stay `false`. We only return `true` if the `Section` has a matching callback.